orchestra/static/orchestra/todos/todo-list.html

```diff
@@ -4,82 +4,84 @@
-              </div>
-              <div class="new-todo__description__datetime">
-                <label>Due</label>
-                <date-picker
-                  date="todoList.newTodoDueDate"
-                  callback="todoList.setTimeOfDate"></date-picker>
-                <time-input
-                  datetime="todoList.newTodoDueDate"
-                  default-hour="18"
-                  ></time-input>
+
+          <form class="new-todo">
+            <div class="new-todo__box new-todo__box-details">
+              <input class="new-todo__description"
+                     type="text"
+                     ng-model="todoList.newTodoDescription"
+                     placeholder="Add a todo item">
+              <div class="pull-right">
+                <div class="new-todo__description__datetime">
+                  <label>Start</label>
+                  <date-picker
+                    date="todoList.newTodoStartDate"
+                    callback="todoList.setTimeOfDate"></date-picker>
+                  <time-input
+                    datetime="todoList.newTodoStartDate"
+                    default-hour="8"
+                    ></time-input>
+                </div>
+                <div class="new-todo__description__datetime">
+                  <label>Due</label>
+                  <date-picker
+                    date="todoList.newTodoDueDate"
+                    callback="todoList.setTimeOfDate"></date-picker>
+                  <time-input
+                    datetime="todoList.newTodoDueDate"
+                    default-hour="18"
+                    ></time-input>
+                </div>
               </div>
             </div>
-          </div>
-          <div class="new-todo__box">
-            <button type="submit"
-               class="btn btn-primary btn-sm edit-save-handle"
-               ng-disabled="!todoList.canAddTodo()"
-               ng-click="todoList.addTodo()">
-              Add
-            </button>
-          </div>
-        </form>
-      </div>
+            <div class="new-todo__box">
+              <button type="submit"
+                 class="btn btn-primary btn-sm edit-save-handle"
+                 ng-disabled="!todoList.canAddTodo()"
+                 ng-click="todoList.addTodo()">
+                Add
+              </button>
+            </div>
+          </form>
 
-      <div class="col-lg-12 col-md-12 col-sm-12">
-          <div class="existing-todos">
-            <todo-checklist
+          <todo-checklist
               title="Todos"
               todos="todoList.todos"
-              show-checked="false"
+              show-skipped="false"
               update-todo="todoList.updateTodo"
+              toggle-skip-todo="todoList.toggleSkipTodo"
               steps="todoList.steps"
               task-slugs="todoList.taskSlugs"
-            ></todo-checklist>
-            <todo-checklist
-              title="Completed"
+          ></todo-checklist>
```

Comment:
Shouldn't there be two todo-checklist's: 1) Todos and 2) Completed?

Reply:
That was the case in the previous implementation. I thought it was redundant to show checked items in a Completed list.

Reply:
Hm... One reason we had a separated completed list was to preserve the CSMs' sanity. It probably doesn't make that much difference. Let's see what @marcua think.

Reply:
I'm cool leaving it out until we hear feedback that it's too overwhelming to have incomplete items there.

I'm also cool adding it now---no strong intuition from expert interviews or anything.

Reply:
Let's not have it then and see whether people want it back.